### DISCUSSION TITLE
DD6A1744-2C07-272E-402B-9AC5EE730912

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#152] [General]
# DD6A1744-2C07-272E-402B-9AC5EE730912

This addon turns Jupiter into a red dwarf star. The moons closest to the planet also undergo changes.

https://celestia.mobi/addon?item=DD6A1744-2C07-272E-402B-9AC5EE730912

**Comment by T512-rgb** [comment]
It's kinda broken
